client/main.lua

```diff
@@ -4,42 +4,43 @@
 
 local config = require 'config.client'
 local functions = require 'client.functions'
-local lockpickDoor, attemptPoliceAlert, isBlacklistedWeapon, isBlacklistedVehicle in functions
+local sharedFunction = require 'shared.functions'
+
+--#region spread functions
+local hasKeys,
+    lockpickDoor,
+    attemptPoliceAlert,
+    isBlacklistedWeapon,
+    isBlacklistedVehicle =
+    functions.hasKeys,
+    functions.lockpickDoor,
+    functions.attemptPoliceAlert,
+    functions.isBlacklistedWeapon,
+    functions.isBlacklistedVehicle
+
+local getHash = sharedFunction.getHash
+--#endregion
 
 -----------------------
 ----   Variables   ----
 -----------------------
 
-local KeysList = {}
-local isTakingKeys = false
-local isCarjacking = false
-local canCarjack = true
-local isHotwiring = false
+local isTakingKeys, isCarjacking, isHotwiring, canCarjack = false, false, false, true
```

Comment:
I prefer the previous syntax as it reduces the chances of me mismatching variables to their values.

Reply:
I also prefer the previous one, however lint throws a false error with it